webviz_core_components/flexbox.py

```diff
@@ -0,0 +1,40 @@
+import copy
+
+import dash_html_components as html
+
+from ._argument_modifier import argument_modifier
+
+
+class FlexBox(html.Div):
+    """Behaves like Div from dash_html_components, but extends that container with
+    flexbox style settings. It also adds necessary flex CSS settings to its children.
+
+    Other style settings are untouched.
+    """
+
+    def __init__(self, *args, **kwargs):
+        def container_style(old_style):
+            style = {} if old_style is None else copy.deepcopy(old_style)
+            style.update({"display": "flex", "flex-wrap": "wrap"})
+            return style
+
+        def update_children_style(children):
+            if children is not None:
+                for child in children:
+                    style = (
+                        copy.deepcopy(child.style) if hasattr(child, "style") else {}
+                    )
+                    style.update({"flex": "auto"})
```

Comment:
It should be possible to have `flex` as input css, e.g. to set variable column width.
Replace with something like:
```python
style["flex"] = style.get("flex", "auto")
```